### DISCUSSION TITLE
Tests import metadata in the DRIIHM OHMi Téssékéré observatory Geonetwork catalogue (Sénégal)

### DISCUSSION THREAD

**Discussion by EmilieLerigoleur** [eblondel/geoflow#270] [Q&A] [answered]
Hello,
I try to import metadata in the Geonetwork catalogue of the DRIIHM OHMi Téssékéré observatory (Sénégal).
Here is the catalogue and an example of metadata file using for the test: https://geo.driihm.fr/geonetwork/ohmi-tes/fre/catalog.search#/metadata/7d71b52b-a58a-422c-80ca-d8f34157dee6

My questions:
1) how to remove the automatically added blue stickers (geometa and ows4r)? (I have no thumbnail)
2) the temporal coverage does not appear, why? (I put 2019)
3) the useLimitation does not appear, why? as a consequence I use otherConstraint parameter for all rights information (= "Contraintes légales")

Many thanks for your help

**Comment by eblondel** [comment, marked as answer]
Hi @EmilieLerigoleur : 
* on question 1:  these logo are probably still defined in your configuration `profile` if you took them from geoflow examples. You can remove them there, you probably have also the `logo` option set to "true" in the ISO 19115 action config, which allows to add config profile logo as thumbnails.
* on question 2: It appears in the XML https://geo.driihm.fr/geonetwork/ohmi-tes/api/records/7d71b52b-a58a-422c-80ca-d8f34157dee6/formatters/xml?approved=true I believe this would be a question for Geonetwork team, why temporal instants (here a single year) do not appear well. You may eventually bypass this by setting it in that way: "2019-01-01/2019-12-31" where geoflow will consider it as period (because of the slash), and - maybe - temporal periods are well handled and displayed in Geonetwork UI
* on question 3: this should work if you use the "use" field in Rights metadata column. Did you use that or "useLimitation". If it is the latter, it will not work and be ignored basically

Let me know if it fixes your issues

**Comment by EmilieLerigoleur** [comment]
- on question 1: ok perfect :)
- on question 2: yes the period works
- on question 3: it works with "use", perfect!
Many thanks

**Reply by eblondel** [reply]
Great!
